src/lib/rest/rest.cpp

```diff
@@ -988,6 +988,7 @@ static int connectionTreat
 
     LM_T(LmtRequest, (""));
     LM_T(LmtRequest, ("--------------------- Serving request %s %s -----------------", method, url));
+    LM_W(("KZ: --------------------- Serving request %s %s -----------------", method, url));
```

Comment:
Strange ... I remember removing that trace ...
I might have put it back and then forgotten it ...

Fixed in c4344c3